integration-tests/runtime-tests/src/generators/crowdloanGenerators/crowdloanRewardGenerator.ts

```diff
@@ -0,0 +1,105 @@
+import { ApiPromise, Keyring, WsProvider } from '@polkadot/api';
+import { KeyringPair } from '@polkadot/keyring/types';
+import minimist from 'minimist';
+import * as R from 'ramda';
+import { args } from '../../utils/args';
+
+/**
+ * Contains all generation methods as static async functions
+ * 
+ * WARN: await Delay can cause problems!
```

Comment:
Would be good to describe what problems :)

Reply:
This will be changed to wait for the extrinsic to be successfully executed